lnd.go

```diff
@@ -660,6 +662,8 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 	// state into Unlocked.
 	interceptorChain.SetWalletUnlocked()
 
+	interceptor.Notifier.NotifyReady(true)
```

Comment:
After the wallet was unlocked the RPC interface might still not be available for a while. So we should perhaps move this call to after `interceptorChain.SetRPCActive()` further below.

Also we should handle the error returned here.

Reply:
In that case yes, we definitely have to move it to get the intended semantics. That's a huge bullet dodged!